doc/electrs-rest-btcd-1.0.0.yaml

```diff
@@ -0,0 +1,936 @@
+      parameters:
+        - name: txid
+          in: path
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Transaction status
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/TxStatus'
+  /tx/{hash}/merkleblock-proof:
+    get:
+      tags:
+        - tx
+      summary: Merkle Inclusing Proof (Bitcoind merkleblock)
+      description: |
+          Returns a Merkle inclusion proof for the transaction using   
+          [bitcoind's merkelblock](https://bitcoin.org/en/glossary/merkle-block) format.  
+          Endpoint Not Compatible with Liquid/Elements based chains.
+      parameters:
+        - name: hash
+          in: path
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Merkle proof
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/MerkleProof'
+  /tx/{hash}/merkle-proof:
+    get:
+      tags: 
+        - tx
+      summary: Merkle Proof (Electrum get-merkle)
+      description: |
+          Returns a Merkle inclusion proof for the transaction  
+          using [Electrum's `get-merkle`](https://electrumx.readthedocs.io/en/latest/protocol-methods.html#blockchain-transaction-get-merkle) format.  
+      parameters:
+        - name: hash
+          in: path
+          required: true
+          schema:
+            $ref: '#/components/schemas/MerkleProof'
+      responses:
+        '200':
+          description: Merkle proof
+          content:
+            application/json:
+              schema:
+                type: object
+  /tx/{hash}/outspend/{index}:
+    get:
+      tags:
+        - tx
+      summary: Returns the spending status of a transaction output
+      parameters:
+        - name: hash
+          in: path
+          required: true
+          schema:
+            type: string
+        - name: index
+          in: path
+          required: true
+          schema:
+            type: integer
+      responses:
+        '200':
+          description: Outspend information
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Outspend'
+  /tx/{hash}/outspends:
+    get:
+      tags: 
+        - tx
+      summary: Get outspend information for all outputs of a transaction
+      parameters:
+        - name: hash
+          in: path
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Outspend information
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Outspend'
+  /broadcast:
+    get:
+      tags: 
+        - tx
+      summary: Broadcast a transaction
+      description: |
```

Comment:
- this description belongs with POST /tx 
- for GET, the tx is specified as a query parameter `?tx=<rawhex>`

Reply:
fixed description to be accurate